XRI3TOXRI2MIGRATIONGUIDE.md

```diff
@@ -0,0 +1,86 @@
+# MRTK3 XRI2 to XRI3 migration guide
+
+## Introduction
+
+Thank you for choosing MRTK as your Mixed Reality framework.  MRTK has been upgraded to follow XRI3 patterns and consume [Unity's XR Interaction Toolkit 3+](https://docs.unity3d.com/Packages/com.unity.xr.interaction.toolkit@3.0/manual/whats-new-3.0.html) package.  The upgrade encompass various changes from the previous MRTK version and we want to share our experience during the upgrade so that the community is aware of the changes that had to be implemented.  We also want to provide an overall roadmap for others to do their own upgrades in their forks or propietary solutions and therefore we created this guide.  We created this guide to empower you and your organization to achieve more in Mixed Reality solutions.  
```

Comment:
encompasses

Reply:
Thx for feedback, implemented in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/839/commits/e14267bbcb572fa95f69535cbc5fc9cbb0a22d00, resolving.